### PR TITLE
bump inline-link to revert styling changes

### LINKED ISSUE #936: Inline-link jumps when hovered on
**Describe the bug**
The bug is described [here](https://github.com/bbc/simorgh/pull/2258#pullrequestreview-258785470). The solution is to either add the padding/margin on default as well as hover or remove it entirely. 

**To Reproduce**

1. On Firefox, create a link that ends very close to the end of the paragraphs column
2. Hover on the inline
3. See the content of the link jump to the next line due to additional padding/margin on hover

**Expected behavior**
The text content shouldn't jump to a new line. 

**Screenshots**
image

image

- Browser: FireFox

### PR DESCRIPTION
**Pull request by pjlee11**
Resolves #936

**Code changes:**

- Bumps `psammead-paragraph` and `psammead-inline-link` to use `psammead-inline-link@1.1.6`
- Re-formats a couple change logs 

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

File: packages/components/psammead-caption/CHANGELOG.md [hunk 1]
```diff
@@ -3,32 +3,33 @@
...
-| 0.2.0 | [PR#318](https://github.com/BBC/psammead/pull/318) Update to new font face and family. |
-| 0.1.6 | [PR#323](https://github.com/BBC/psammead/pull/323) Update readme storybook badge |
-| 0.1.5 | [PR#264](https://github.com/BBC/psammead/pull/319) Resolving package-lock issues. |
-| 0.1.4 | [PR#264](https://github.com/BBC/psammead/pull/264) Resolving package-lock issues. |
-| 0.1.3 | [PR#245](https://github.com/BBC-News/psammead/pull/245) Ensures documentation consistent across component packages. |
-| 0.1.2 | [PR#231](https://github.com/BBC-News/psammead/pull/231) Add link to Storybook to README |
-| 0.1.1 | [PR#227](https://github.com/BBC-News/psammead/pull/227) Replace @bbc/gel-constants and @bbc/gel-foundations-styled-component with [@bbc/gel-foundations in Psammead](https://github.com/BBC-News/psammead/issues/226). |
-| 0.1.0 | [PR#186](https://github.com/BBC-News/psammead/pull/186) Create initial package, pulled in from [Simorgh](https://github.com/BBC-News/simorgh). |
+| 1.1.12  | [PR#xxx](https://github.com/bbc/psammead/pull/xxx) Bump psammead-inline-link to revert styling changes |
```
Comment (pjlee11) [inline packages/components/psammead-caption/CHANGELOG.md]:
```suggestion
| 1.1.12  | [PR#1019](https://github.com/bbc/psammead/pull/1019) Bump psammead-inline-link to revert styling changes |
```

File: packages/components/psammead-paragraph/CHANGELOG.md [hunk 2]
```diff
@@ -3,7 +3,8 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
-| 1.0.9 | [PR#783](https://github.com/bbc/psammead/pull/783) Update to latest psammead-test-helpers. Update snapshots. |
+| 1.0.10  | [PR#xxx](https://github.com/bbc/psammead/pull/xxx) Bump psammead-inline-link to revert styling changes |
```
Comment (pjlee11) [inline packages/components/psammead-paragraph/CHANGELOG.md]:
```suggestion
| 1.0.10  | [PR#1019](https://github.com/bbc/psammead/pull/1019) Bump psammead-inline-link to revert styling changes |
```

**Comment by pjlee11** [comment]
Caption
image

Paragraph 
image

**Review by dr3 — APPROVED** [review]

**Review by 12 — APPROVED** [review]
image

LGTM

**Pull request merged**